openclsim/core.py

```diff
@@ -1298,7 +1298,7 @@ def determine_route(self, origin, destination):
         geom = nx.get_node_attributes(self.env.FG, "geometry")
 
         for node in geom.keys():
-            if origin.x == geom[node].x and origin.y == geom[node].y:
+            if abs(origin.x - geom[node].x)<0.00000001 and abs(origin.y - geom[node].y)<0.00000001:
```

Comment:
Could you solve it with [numpy.isclose](https://docs.scipy.org/doc/numpy/reference/generated/numpy.isclose.html)? That is a more elegant solution.
And remember the Black formatting.